packages/medusa/src/api/routes/admin/payment-collections/index.ts

```diff
@@ -0,0 +1,96 @@
+import { Router } from "express"
+import "reflect-metadata"
+import middlewares, {
+  transformBody,
+  transformQuery,
+} from "../../../middlewares"
+import { AdminCreatePaymentCollectionRequest } from "./create-payment-collection"
+import OrderEditingFeatureFlag from "../../../../loaders/feature-flags/order-editing"
+import { isFeatureFlagEnabled } from "../../../middlewares/feature-flag-enabled"
+
+import { GetPaymentCollectionsParams } from "./get-payment-collection"
+import { AdminUpdatePaymentCollectionRequest } from "./update-payment-collection"
+
+const route = Router()
+
+export default (app, container) => {
+  app.use(
+    "/payment-collections",
+    isFeatureFlagEnabled(OrderEditingFeatureFlag.key),
+    route
+  )
+
+  route.get(
+    "/:id",
+    transformQuery(GetPaymentCollectionsParams, {
+      defaultFields: defaultPaymentCollectionFields,
+      defaultRelations: defaulPaymentCollectionRelations,
+      isList: false,
+    }),
+    middlewares.wrap(require("./get-payment-collection").default)
+  )
+
+  route.post(
+    "/",
+    transformBody(AdminCreatePaymentCollectionRequest),
+    middlewares.wrap(require("./create-payment-collection").default)
+  )
+
+  route.post(
+    "/:id",
+    transformBody(AdminUpdatePaymentCollectionRequest),
+    middlewares.wrap(require("./update-payment-collection").default)
+  )
+
+  route.delete(
+    "/:id",
+    middlewares.wrap(require("./delete-payment-collection").default)
+  )
+
+  route.post(
+    ":id/payments/capture",
+    middlewares.wrap(require("./capture-all-payment-collection").default)
+  )
+
+  route.post(
+    "/payments/:payment_id/capture",
+    middlewares.wrap(require("./capture-payment").default)
+  )
+
+  route.post(
+    ":id/payments/refund",
+    middlewares.wrap(require("./refund-all-payment-collection").default)
+  )
+
+  route.post(
+    "/payments/:payment_id/refund",
+    middlewares.wrap(require("./refund-payment").default)
+  )
```

Comment:
I think we should move these to a separate domain that is specific to payments so that we can consume these completely independently from payment collections:

```
GET /admin/payments
GET /admin/payments/:id
POST /admin/payments/:id/capture
POST /admin/payments/:id/refund
```